### ISSUE TITLE
Integrate airport departures / arrivals ? 

### ISSUE THREAD

**Issue by petzlux** [TfLlu/TfL-API#37]

**Comment by pukkatea** [comment]
is there an endpoint?

only found this one: https://www.lux-airport.lu/index.php?flightStatus_action=getFlightStatus

**Comment by petzlux** [comment]
We will check but sure is interesting data, and a rather clean JSON :-)

**Comment by petzlux** [comment]
Check for permission/licence

**Comment by pukkatea** [comment]
https://api.tfl.lu/v1/airport/arrivals
https://api.tfl.lu/v1/airport/departures

**Comment by petzlux** [comment]
Hmm, arrivals end point output cant find flight number that match whats shown on website ?
![screen shot 2017-03-24 at 09 54 29](https://cloud.githubusercontent.com/assets/2540428/24286982/e8f852f6-1077-11e7-8dd2-00aee97b6687.png)

**Comment by petzlux** [comment]
Did you manage to find the problems ? Can this be merged ?

**Comment by pukkatea** [comment]
I had a look at this again, and it's not a bug (it's a feature 😋)

It was (and still) hard to work with undocumented API endpoints that have no clear way of being understood.
Here's what happened:
looking at the data, we found that the best way to remove flights from the past was to look at the 'expected_at' timestamp.

According to the following screenshot I took at 11.43.50 today, the data is inconsistent and not updated as often as their status is.

<img width="962" alt="screen shot 2017-04-05 at 11 34 50 am" src="https://cloud.githubusercontent.com/assets/25056651/24699510/7a592c7c-19f4-11e7-9e88-da020dcea1a9.png">

Next steps:
we'll not rely on the 'expected_at' timestamp but check for the status not to be 'take off' statusCode (10) of 'arrived' statusCode (13)

But as @grischard's [question](https://data.public.lu/en/datasets/arrivees-et-departs-de-laeroport-de-luxembourg/#discussion-58e3e665111e9b483ec318a4) is still unanswered, this is just an assumption as the API has no documentation.
